addon/services/-observer-admin.js

```diff
@@ -62,10 +62,10 @@ export default class ObserverAdmin extends Service {
 
```

Comment:
r/`let { root = window } = observerOptions;`/`let { root } = observerOptions;`

Root will just default to the document viewport if null

Reply:
Sry I was wrong.  `window` is needed for the weakmap.